programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md

```diff
@@ -8,7 +8,8 @@
 - Description:
 - Expected Outcome:
 - Recommended Skills:
-- Mentor(s): Mentor Name (@mentor_github, mentor@email.addy)
+- Mentor(s):
+  - Mentor Name (@mentor_github, mentor@email.addy - please use the same email address as you use on the [LFX Mentorship Platform](mentorship.lfx.linuxfoundation.org/))
```

Comment:
```suggestion
  - Mentor Name (@mentor_github, mentor@email.addy) - please use the same email address as you use on the LFX Mentorship Platform at https://mentorship.lfx.linuxfoundation.org
```

Since this is a code block in the markdown file, the links won't render and the text becomes unreadable (example: https://github.com/cncf/mentoring/blob/6f70f902f3bd08568fbae525ea5bd73b47504acd/programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md)

Reply:
Good catch, thx